thirdparty/CMakeLists.txt

```diff
@@ -191,9 +191,17 @@ endif()
 #
 # nlohmann json
 #
-if(ENABLE_SAMPLES OR ENABLE_OV_PDPD_FRONTEND OR ENABLE_OV_ONNX_FRONTEND OR ENABLE_OV_TF_FRONTEND)
-    add_subdirectory(nlohmann_json EXCLUDE_FROM_ALL)
-    openvino_developer_export_targets(COMPONENT openvino_common TARGETS nlohmann_json)
+if(ENABLE_SAMPLES OR NGRAPH_PDPD_FRONTEND_ENABLE OR NGRAPH_ONNX_FRONTEND_ENABLE OR NGRAPH_TF_FRONTEND_ENABLE)
+    function(build_json_libs)
+        set(JSON_Install ON)
```

Comment:
looks like `JSON_Install` is not needed here

Reply:
I will re-check

Reply:
looks like it's required by nlohmann_json_schema_validator 
![image](https://user-images.githubusercontent.com/21181928/145357365-89fe60ef-807d-4fc2-9938-002bed9a700d.png)